config/environments/development.rb

```diff
@@ -71,6 +71,8 @@
     Bullet.bullet_logger = true
     Bullet.rails_logger = true
   end
+
+  config.deploy_wip_pipelines = true
```

Comment:
This sets a config option similar to Limber but it needs to be used by something. The loader you have written needs to act according to this option for enabling WIPs. At the moment, the loader is trying to enable all WIPS by returning all without any question.

Reply:
In one of our meetings, I think we have talked about a better naming to avoid limiting the semantics to **pipelines** in Sequencescape context.

Reply:
This PR needs to be accompanied by a deployment project PR to set the option value to `true` for UAT environment and `false` for production and training.

The piece of code using this option must also make sure that it does not load features unless it is explicitly set. This is very important to prevent loading WIPs in training and production.

Reply:
> This sets a config option similar to Limber but it needs to be used by something. The loader you have written needs to act according to this option for enabling WIPs. At the moment, the loader is trying to enable all WIPS by returning all without any question.

We either to set it loading all WIPs, or none of the WIPs, is that right?

Reply:
Yes, if you have to test it on Training. The DB is reset every Saturday from Production DB.